ember-rails.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |s|
   s.add_dependency "railties", [">= 3.1"]
   s.add_dependency "active_model_serializers"
   s.add_dependency "barber", [">= 0.6.0"]
+  s.add_dependency "es6_module_transpiler-rails"
```

Comment:
ember-cli uses [esperanto](https://github.com/ember-cli/ember-cli/blob/v0.1.11/package.json#L90).
On the other hand, blueprints uses [6to5](https://github.com/ember-cli/ember-cli/blob/v0.1.11/blueprints/app/files/package.json#L25).

What is the difference of both of them?